index.html

```diff
@@ -1,67 +1,345 @@
+                    <span>2015</span>
+                  </div>
+                </div>
+              </div>
+              <div class="primary-text-4">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              </div>
+                <div class="tags">
+                  <ul>
+                    <li>
+                      <div class="tag-html">
+                        <span>html</span>
+                      </div>
+                    </li>
+                    <li>  
+                      <div class="tag-css">
+                        <span>css</span>
+                      </div>
+                    </li>
+                    <li>
+                      <div class="tag-js">
+                        <span>javaScript</span>
+                      </div>
+                    </li>
+                  </ul>
+                </div>
+              <div class="action">
+                <div class="see-project">
+                  <button>See Project</button>
+                </div>
+              </div>
+            </div>
+          </div>
         </div>
-      </div>
-      <div class="primary-text">
-        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-      </div>
-
-    </section>
-  </main>
-
-</body>
-</html>
\ No newline at end of file
+        <section class="about-myself">
+          <div class="top">
+            <div class="primary-text-5">
+              <h1>About <br>
+              Myself</h1>
+            </div>
+              <p class="primary-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="social-media">
+              <div class="supporting-text">
+                <span>LET'S CONNECT!</span>
+              </div>
+            </div>
+            <div class="social-media-icons">
+              <ul class="social-media-icons-list">
+                <li><a href="#"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
+                <li><a href="#"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
+                <li><a href="#"><img src="assets/m.svg" alt="Medium" width="20" height="16"></a></li>
+                <li><a href="#"><img src="assets/git.svg" alt="GitHub" width="18" height="20"></a></li>
+                <li><a href="#"><img src="assets/finger.svg" alt="Peace" width="15" height="20"></a></li>
+              </ul>
+            </div>
+            <div class="normal-button-secondary">
+              <div class="get-my-resume">
+                <button>Get my resume</button>
+              </div>
```

Comment:
- [ ] The earlier suggestions ☝️  I gave about your buttons apply to this one, too